frontend/projects/abtesting/src/app/shared/components/delete/delete.component.scss

```diff
@@ -0,0 +1,20 @@
+.dialog {
+    .name {
```

Comment:
@YashilDepani Can you please fix the indentation here?

Reply:
code updated